ratelimit/utils.py

```diff
@@ -4,9 +4,16 @@
 import zlib
 
 from django.conf import settings
-from django.core.cache import get_cache
+try:
+    from django.core.cache import caches
+except ImportError:
+    from django.core.cache import get_cache as caches
```

Comment:
We can avoid the awkward, and too-long, line at 137 by reversing this, and doing something like

```python
try:
    from django.core.cache import caches
    def get_cache(name):
        return caches[name]
except ImportError:
    from django.core.cache import get_cache
```
Then just calling `get_cache` on 137 (and anywhere else we'd need to in the future).